### ISSUE TITLE
[fxa-content-server] Fix all SCSS warnings

### ISSUE THREAD

**Issue by LZoog** [mozilla/fxa#2239]
Relates to #2088, but there are many more warnings than were spat out in that log. There are about 70 in `_settings.scss` alone. Does everyone have `scss-lint` working in their editor? I didn't until just a few days ago and now I'm seeing warnings like this everywhere.

This would be a fairly large effort across all of our SCSS files because we frequently style by ID. This could be an opportunity to refactor a lot of the front-end styles to better use variables and mixins.

It would also be good to go through the payments server SCSS.

![image](https://user-images.githubusercontent.com/13018240/63195417-1edcd000-c038-11e9-8db5-b94be34b4700.png)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-282)
┆Issue Number: FXA-282

**Comment by AlbionaHoti** [comment]
@clouserw I am interested to work on this task, can I continue?

**Comment by shane-tomlinson** [comment]
> @clouserw I am interested to work on this task, can I continue?

You sure can!

**Comment by clouserw** [comment]
@AlbionaHoti are you still working on this?  Do you need help?

**Comment by byrw** [comment]
Can I work on it?

Hey @LZoog, I have some questions about this issue.

**Questions** 

1. One of the sccs-lint rules is that !mportant should not be used. Considering my inexperience with the codebase, should I try to work on this? 
2. Another rule is to avoid qualify attribute selectors with an element. So in this example, scss-lint will give a warning:
`
.fxa-radio {
  html[dir='ltr'] & {
    text-align: left;
  }
`
How do I work around this? Should I try to create instead a class as substitute of dir?

**Comment by LZoog** [comment]
@byrw our team is going through and reviewing what we've marked as `good first issue`.

This is probably not a good one at the moment to grab. We need to address all of the warnings before implementing #2088, but we're about to propose a rearchitecture of our CSS and at that point in time it would be ideal to address these linting issues at the same time. We've removed the associated labels.

To answer your questions though:
- What I'd do would be to locate the element on the page where that style is being applied, take off the `!important`, and see what is overriding the property in dev tools. The properties we have marked as `!important` likely just need to be tied to higher selector specificity.
- I think that rule is complaining about having the `html` piece of `html[dir='rtl']`. `[dir='rtl']` is the 'attribute selector' and it's saying the selector could be styled more simply as `[dir='rtl'] & {` instead of `html[dir='rtl]`.

... we will certainly approve and merge any PRs that address any of the warnings seen here since it's going to be a few more months before we refactor this section to prioritizing work elsewhere, but time spent on other `good first issue` issues would be more prudent. 🙂
